rules/linux/at_command.yml

```diff
@@ -0,0 +1,26 @@
+title: Scheduled Task/Job At
+id: d2d642d7-b393-43fe-bae4-e81ed5915c4b
+status: stable
+description: Detects the use of at/atd
+author: Ömer Günal, oscd.community
+date: 2020/10/06
+references:
+    - https://attack.mitre.org/techniques/T1053/001/
```

Comment:
Could you please remove a link to ATT&CK (as it is already in the tags) and put there a reference to the relevant [Atomic Red Team test](https://github.com/redcanaryco/atomic-red-team/blob/master/atomics/T1053.001/T1053.001.md)?

Reply:
Hello @omergunal! Could you please fix that?